models/Department.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema
+
+const departmentSchema = new Schema({
+    id: {
+        type: String,
+        require: true
+    },
+    name: {
+        type: String,
+        require: true
+    },
+    description: {
+        type: String,
+        require: true
+    },
+    created_at: {
+        type: String,
+        require: true
+    },
+    updated_at: {
+        type: String,
+        require: true
+    },
+    employees: {
+        type: Array,
+        require: true
+    }
```

Comment:
you should define the relations
https://medium.com/@brandon.lau86/one-to-many-relationships-with-mongodb-and-mongoose-in-node-express-d5c9d23d93c2